tests/checkers/unittest_imports.py

```diff
@@ -40,6 +41,9 @@ def test_relative_beyond_top_level(self) -> None:
             self.checker.visit_importfrom(module.body[2].body[0])
 
     @staticmethod
+    @pytest.mark.xfail(
+        reason="epylint manipulates cwd; these tests should not be using epylint"
```

Comment:
Shouldn't we fix it or remove it ? What's the benefit of keeping xfailed tests ?

Reply:
Yeah, I just didn't want to block this PR on resolving that. All of the tests in that file probably shouldn't be using epylint. Would you like me to open an issue?

Reply:
Yeah let's open an issue, I'm not sure emacs is used a lot so it's probably not worth a fix, no one read the emacs documentation at least.

Reply:
#7252

Reply:
Thank you !